src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -106,16 +108,17 @@ private static Volunteer createEditedVolunteer(Volunteer volunteerToEdit,
                                                    EditVolunteerDescriptor editVolunteerDescriptor) {
         assert volunteerToEdit != null;
 
-        VolunteerId volunteerId = volunteerToEdit.getVolunteerId();
         Name updatedName = editVolunteerDescriptor.getName().orElse(volunteerToEdit.getName());
+        VolunteerId updatedVolunteerId = editVolunteerDescriptor.getVolunteerId()
```

Comment:
VolunteerId volunteerId = volunteerToEdit.getVolunteerId();

Probably can keep that line since volunteer NRIC should not be changed.